src/cargo/ops/cargo_new.rs

```diff
@@ -399,20 +424,22 @@ pub fn new(opts: &NewOptions, config: &Config) -> CargoResult<()> {
         )
     }
 
+    let kind = match opts.kind {
+        NewProjectKind::Bin => NewProjectKind::Bin,
+        NewProjectKind::Auto => NewProjectKind::Bin,
+        _ => NewProjectKind::Lib,
+    };
```

Comment:
These can be deleted If removing `NewProjectKind::Auto`.

Reply:
Done